client/galaxy/scripts/utils/uploadbox.js

```diff
@@ -97,7 +97,7 @@
         }, false);
 
         // send request
-        Galaxy.emit.debug('uploadbox::uploadpost()', 'Posting following data.', cnf);
+        Galaxy.logger.emit('debug', 'uploadbox::uploadpost()', [ 'Posting following data.', cnf ]);
```

Comment:
These two lines are equivalent and I personally prefer the above version. It is slightly shorter and I think more intuitive. However, if  we prefer to access the logger object first, specify the target as string and add Array brackets, we should just do this for all log statements and remove the emit helper function.